timer/api/timer.go

```diff
@@ -211,3 +272,13 @@ func (r *TimerRecord) Clone() *TimerRecord {
 	cloned.TimerSpec = *r.TimerSpec.Clone()
 	return &cloned
 }
+
+// ValidateTimeZone validates the TimeZone field.
+func ValidateTimeZone(tz string) error {
+	if tz != "" && !strings.EqualFold(tz, TimeZoneTiDB) {
+		if _, ok := timeutil.ParseTimeZone(tz); !ok {
+			return errors.Errorf("Invalid timezone '%s'", tz)
```

Comment:
IMO it's better to use `ErrUnknownTimeZone`? `errors.Errorf` gives 'unknown error code' which we should always avoid when possible.

And, is there any specific reason we need to change the return value of `ParseTimeZone` from `error` to `bool`?

Reply:
`ParseTimeZone` returns bool is easy to write without introducing another `errors.go` file... But anyway it not very hard to do it, I'll update the PR.